doc/ledger3.texi

```diff
@@ -512,7 +512,7 @@ Ledger will generate:
 
 @smallexample @c output:66E3A2C
 10-Dec-01 Checking balance      Assets:Checking          $ 1,000.00   $ 1,000.00
-                                Equit:Opening Balances  $ -1,000.00            0
+                                Equity:Opening Balances  $ -1,000.00            0
```

Comment:
This MUST NOT be merged.

As explained in the beginning of ledger3.texi, the examples provided in documentation are validated with ledger command. Here, by adding a 'y', you are modifying the ledger output. As DocTests.py is checking that ledger3.texi is the same as ledger output, this fails...